test/integration-test.mk

```diff
@@ -26,7 +26,15 @@ clean-integration-test:
 
 clean: clean-integration-test
 
-test-integration: ensure-kubebuilder-tools
-	go test -c ./test/integration
-	./integration.test -ginkgo.slowSpecThreshold=15 -ginkgo.v -ginkgo.failFast
+test-kube-integration: ensure-kubebuilder-tools
+	go test -c ./test/integration/kube -o ./kube-integration.test
+	./kube-integration.test -ginkgo.slowSpecThreshold=15 -ginkgo.v -ginkgo.failFast
+.PHONY: test-kube-integration
+
+test-cloudevents-integration: ensure-kubebuilder-tools
+	go test -c ./test/integration/cloudevents -o ./cloudevents-integration.test
+	./cloudevents-integration.test -ginkgo.slowSpecThreshold=15 -ginkgo.v -ginkgo.failFast
+.PHONY: test-cloudevents-integration
+
+test-integration: test-kube-integration test-cloudevents-integration
```

Comment:
created separate folder for cloudevents integration test.
make sense? @qiujian16 @skeeey

Reply:
yeah, we may do some configuration like this https://github.com/open-cluster-management-io/ocm/blob/main/.github/workflows/cloudevents-integration.yml#L5